modules/govuk_jenkins/templates/jobs/smokey.yaml.erb

```diff
@@ -30,9 +30,30 @@
           auth-token: <%= @environment_variables['SECOND_LINE_SLACK_AUTH_TOKEN']%>
           build-server-url: <%= @slack_build_server_url %>
           room: <%= @slack_room %>
-      - email:
+      - email-ext:
           recipients: govuk-ci-notifications@digital.cabinet-office.gov.uk
-          send-to-individuals: true
+          failure: true
+          presend-script: |
```

Comment:
It may be possible to add it as a library like [we do here](https://github.com/alphagov/govuk-puppet/blob/master/modules/govuk_jenkins/manifests/pipeline.pp#L24-L30).